test/accessibility/apps/management.ts

```diff
@@ -21,83 +21,83 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const testSubjects = getService('testSubjects');
 
   describe('Management', () => {
-    before(async () => {
-      await esArchiver.loadIfNeeded('test/functional/fixtures/es_archiver/logstash_functional');
-      await kibanaServer.importExport.load('test/functional/fixtures/kbn_archiver/discover');
-      await kibanaServer.uiSettings.update({
-        defaultIndex: 'logstash-*',
-      });
-      await PageObjects.settings.navigateTo();
-    });
-
-    after(async () => {
-      await kibanaServer.importExport.unload('test/functional/fixtures/kbn_archiver/discover');
-      await esArchiver.unload('test/functional/fixtures/es_archiver/logstash_functional');
-    });
-
     it('main view', async () => {
       await a11y.testAppSnapshot();
```

Comment:
All tests have to have a `navigate` call somewhere near the beginning.  You currently can't run this test by itself, only after some other test has already run and navigated somewhere into Kibana.

Reply:
Fixed.